test/forcing/ForcingsEngineLumpedDataProvider_Test.cpp

```diff
@@ -0,0 +1,132 @@
+#include <gtest/gtest.h>
+
+#ifndef NGEN_LUMPED_CONFIG_PATH
+#error "Lumped config file path not defined! Set `-DNGEN_LUMPED_CONFIG_PATH`"
+#endif
+
+#include <NGenConfig.h>
+#if NGEN_WITH_MPI
+#include <mpi.h>
+#endif
+
+#include <forcing/ForcingsEngineLumpedDataProvider.hpp>
+#include <forcing/AorcForcing.hpp>
+#include <utilities/python/InterpreterUtil.hpp>
+
+#include <string>
+
+struct ForcingsEngineLumpedDataProviderTest
+  : public testing::Test
+{
+    using provider_type = data_access::ForcingsEngineLumpedDataProvider;
+
+    ForcingsEngineLumpedDataProviderTest()
+    {
+        #if NGEN_WITH_MPI
+        MPI_Comm_size(MPI_COMM_WORLD, &mpi_.size);
+        MPI_Comm_rank(MPI_COMM_WORLD, &mpi_.rank);
+        #endif
+    }
+
+    static constexpr const char* config_file = NGEN_LUMPED_CONFIG_PATH;
```

Comment:
Does this actually end up expanding to a string literal like intended? I guess if CMake ensures that the argument as written gets shell-escaped so that the macro definition includes the `"` around the name, then it would. That's just surprising to me, now that I noticed it.

Reply:
Yeah, the compilation db after configuring contains the following for this file:

```
-DNGEN_LUMPED_CONFIG_PATH=\\\"/path/to/ngen/build/test/config_aorc.yml\\\"
```

It may not be the best approach, but nothing else immediately comes to my mind as an alternative. But, if you prefer a different approach, I'm open to it!